modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="nw-ovn-kubernetes-migration_{context}"]
-= Migrating to the OVN-Kubernetes default CNI network provider
+= Migrating to the OVN-Kubernetes network plugin
```

Comment:
This is a global comment but I'm putting here because I see mixed use of "plugin" and "plug-in". According to the RH Supplementary SG, it is advised not to use "plug-in" in new content unless you're updating an area that already uses the hyphenated form. I didn't dig into the contextuals of the areas that use plug-in, so maybe you used that because its used elsewhere in the docset. Just wanted to flag that I noted mixed use and make sure its supposed to be that way. 
https://redhat-documentation.github.io/supplementary-style-guide/#plugin

Reply:
@skrthomas, yeah, I noticed this a month ago. All new instances are correct. I can go back and fix the rest after, so I reduce the odds of merge conflicts for this.